File: 01. SourceCode/Android/ChildProfile/app/src/main/java/nts/childprofile/model/UserUploadImageModel.java
package nts.childprofile.model;

public class UserUploadImageModel {
    private String Content;
    private String UserId;

    public UserUploadImageModel(String content, String userId) {
        Content = content;
        UserId = userId;
    }

    public String getContent() {
        return Content;
    }

    public void setContent(String content) {
        Content = content;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }
}

File: 01. SourceCode/IOS/childprofile/childprofile/Models/ShareImage/UserUploadImageModel.swift
//
//  UserUploadImageModel.swift
//  childprofile
//
//  Created by Thanh Luu on 1/27/19.
//  Copyright © 2019 childfund. All rights reserved.
//

import Foundation

struct UserUploadImageModel: Encodable {
    var content: String
    var userId: String
    
    enum CodingKeys: String, CodingKey {
        case content = "Content"
        case userId = "UserId"
    }
}
